index.js

```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
 // Read the history file and parse the data
 
 function readFile() {
-  const array = fs.readFileSync("testHistory.txt").toString().split("\n");
+  const homeDir = os.homedir();
+  const historyPath = path.join(homeDir, ".bash_history");
+  const possiblePath = [
+    path.join(homeDir, ".bash_history"),
+    path.join(homeDir, ".zsh_history"),
+    path.join(homeDir, ".history"),
+  ];
 
-  return array;
+  // need to find the biggest history file
+
+  let biggestHistorySize = 0;
+  let biggestHistoryPath;
+
+  for (const path of possiblePath) {
```

Comment:
This might be better as a SINGLE function outside of this function. This function is doing "too much work".